packages/event-handler/package.json

```diff
@@ -77,6 +77,16 @@
         "types": "./lib/esm/rest/index.d.ts",
         "default": "./lib/esm/rest/index.js"
       }
+    },
+    "./experimental-rest/middleware": {
```

Comment:
We need to add the new path to the `typeVersions` below as well, or when you actually use it, this happens:
<img width="826" height="167" alt="Screenshot 2025-09-17 at 00 04 44" src="https://github.com/user-attachments/assets/181be4b5-6099-4229-91f2-f90a163f4fc4" />